Source/VCGeneration/Counterexample.cs

```diff
@@ -134,7 +134,7 @@ public void AddCalleeCounterexample(Dictionary<TraceLocation, CalleeCounterexamp
     }
 
     // Looks up the Cmd at a given index into the trace
-    public Cmd getTraceCmd(TraceLocation loc)
+    public Cmd GetTraceCmd(TraceLocation loc)
```

Comment:
Is it OK to rename these public methods?

Reply:
I think so, GPUVerify has said they're OK with minor API changes. The other C# customer AFAIK is Corral but I think similarly small changes are OK.